tests/10-vcdm2.js

```diff
@@ -97,20 +98,25 @@ describe('Verifiable Credentials Data Model v2.0', function() {
       return result;
     }
 
-    async function verifyVp(vp) {
+    async function verifyVp(vp, options = {checks: []}) {
       const body = {
         verifiablePresentation: vp,
-        options: {
-          checks: ['proof'],
-        }
+        options
       };
       const result = await post(vpVerifier, body);
-      if(result.errors.length) {
+      if(result?.errors?.length) {
         throw result.errors[0];
       }
       return result;
     }
 
+    const challenge = 'z19pnWMzs2Ub452WW9BLBJP9h';
```

Comment:
Generate a new challenge each time to ensure implementations are not hard coding the challenge.

Reply:
I used `Date.now()` as the challenge: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/26/commits/a6158ad9e20d07bccb3070cc728b222d1be0e15f

As a String not a number.

Reply:
No, don't use Date.now() -- use a 128-bit random number and base64url encode the value w/ a multibase prefix of 'u'.

Reply:
Using a base64 encoded 128 byte number here: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/26/commits/137554f8c18ae1a3e861ee74af0aa9c231f1345c

Reply:
added the `u` multibase prefix here: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/26/commits/5cd3784f7eaa087ad38f82e3fede0b439faa8686